pkg/controller/autoimport/autoimport_controller.go

```diff
@@ -92,20 +94,22 @@ func (r *ReconcileAutoImport) Reconcile(ctx context.Context, request reconcile.R
 
 	if _, autoImportDisabled := managedCluster.Annotations[apiconstants.DisableAutoImportAnnotation]; autoImportDisabled {
 		// skip if auto import is disabled
+		reqLogger.Info("Auto import is disabled", "managedCluster", managedCluster.Name)
 		return reconcile.Result{}, nil
+	} else {
+		reqLogger.Info("Auto import is enabled", "managedCluster", managedCluster.Name)
```

Comment:
this will generate a lot of logs, might add the log level

Reply:
Done